### PR TITLE
update(cassandra): eol dates for 4.0 and 4.1 versions

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/DOC-867

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

File: docs/platform/reference/eol-for-major-versions.md [hunk 1]
```diff
@@ -93,13 +93,11 @@ after they are made available on the Aiven platform.
...
 ### Aiven for Apache Cassandra® {#h_0f2929c770}
 
-Starting with v4, Aiven for Cassandra® major versions will reach EOL six
-months after they are made available on the Aiven platform.
-
 | Version | Aiven EOL  | Service creation supported until | Service creation supported from |
 | ------- | ---------- | -------------------------------- | ------------------------------- |
 | 3       | 2022-07-27 | 2022-04-27                       | 2018-11-08                      |
-| 4       | N/A        | N/A                              | 2021-12-09                      |
+| 4.0     | 2024-09    | N/A                              | 2021-12-09                      |
```
Comment (alanfranz) [inline docs/platform/reference/eol-for-major-versions.md:99]: we should use 2024-09-30 and 2025-09-30 - I think we use a well-defined day in all of our products.

Comment (alanfranz) [reply docs/platform/reference/eol-for-major-versions.md:99]: Now we're discussing a "service creation supported until" date for 4.0 as well.

**Review by alanfranz — APPROVED** [review]

**Comment by alanfranz** [comment]
@wojcik-dorota LGTM but I don't have merge rights on this.

**Review by staceysalamon-aiven — APPROVED** [review]
LGTM

**Review by ArthurFlag — APPROVED** [review]

**Pull request merged**